pkg/service/model_build_load_balancer.go

```diff
@@ -232,42 +232,94 @@ func (t *defaultModelBuildTask) resolveLoadBalancerSubnets(ctx context.Context,
 }
 
 func (t *defaultModelBuildTask) buildLoadBalancerAttributes(_ context.Context) ([]elbv2model.LoadBalancerAttribute, error) {
-	var attrs []elbv2model.LoadBalancerAttribute
-	accessLogEnabled := t.defaultAccessLogS3Enabled
-	bucketName := t.defaultAccessLogsS3Bucket
-	bucketPrefix := t.defaultAccessLogsS3Prefix
-	if _, err := t.annotationParser.ParseBoolAnnotation(annotations.SvcLBSuffixAccessLogEnabled, &accessLogEnabled, t.service.Annotations); err != nil {
+	defaultAttribs := t.getDefaultAnnotationLbAttributes()
+	configMapAttribs, err := t.getConfigMapLbAttributes()
+	if err != nil {
 		return []elbv2model.LoadBalancerAttribute{}, err
 	}
-	if accessLogEnabled {
-		t.annotationParser.ParseStringAnnotation(annotations.SvcLBSuffixAccessLogS3BucketName, &bucketName, t.service.Annotations)
-		t.annotationParser.ParseStringAnnotation(annotations.SvcLBSuffixAccessLogS3BucketPrefix, &bucketPrefix, t.service.Annotations)
-	}
-	crossZoneEnabled := t.defaultLoadBalancingCrossZoneEnabled
-	if _, err := t.annotationParser.ParseBoolAnnotation(annotations.SvcLBSuffixCrossZoneLoadBalancingEnabled, &crossZoneEnabled, t.service.Annotations); err != nil {
+	specificAttribs, err := t.getAnnotationSpecificLbAttributes()
+	if err != nil {
 		return []elbv2model.LoadBalancerAttribute{}, err
 	}
+	mergedAttribs := mergeAttributeMaps(defaultAttribs, mergeAttributeMaps(configMapAttribs, specificAttribs))
+	return mkAttributesSliceFromConfigMap(mergedAttribs), nil
+}
+
+func mergeAttributeMaps(firstMap map[string]string, secondMap map[string]string) map[string]string {
+	for key, value := range secondMap {
+		firstMap[key] = value
+	}
+	return firstMap
+}
+
+func mkAttributesSliceFromConfigMap(configMapAttrs map[string]string) []elbv2model.LoadBalancerAttribute {
+	attributes := make([]elbv2model.LoadBalancerAttribute, 0, len(configMapAttrs))
+	keys := make([]string, 0, len(configMapAttrs))
+	for k := range configMapAttrs {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	for _, attrKey := range keys {
+		attributes = append(attributes, elbv2model.LoadBalancerAttribute{
+			Key:   attrKey,
+			Value: configMapAttrs[attrKey],
+		})
+	}
+	return attributes
+}
 
-	attrs = []elbv2model.LoadBalancerAttribute{
-		{
-			Key:   lbAttrsAccessLogsS3Enabled,
-			Value: strconv.FormatBool(accessLogEnabled),
-		},
-		{
-			Key:   lbAttrsAccessLogsS3Bucket,
-			Value: bucketName,
-		},
-		{
-			Key:   lbAttrsAccessLogsS3Prefix,
-			Value: bucketPrefix,
-		},
-		{
-			Key:   lbAttrsLoadBalancingCrossZoneEnabled,
-			Value: strconv.FormatBool(crossZoneEnabled),
-		},
+func (t *defaultModelBuildTask) getDefaultAnnotationLbAttributes() map[string]string {
```

Comment:
lets not specify default attributes, so the AWS defaults get used, or the controller preserves existing attributes.